api/v1alpha1/webconsolerequest_types.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type SubjectReference struct {
```

Comment:
Is the SubjectReference needed anymore since this particular request will always be for a VM resource?